bower.json

```diff
@@ -1,13 +1,15 @@
 {
-	"name": "o-ft-footer",
-	"main": ["main.scss"],
-	"ignore": [],
-	"dependencies": {
-		"o-colors": "^2.3.18",
-		"o-ft-typography": "^1.12.1",
-		"o-grid": "^2.1.5",
-		"o-assets": "^0.4.0",
-		"o-fonts": "^1.4.1",
-		"o-hoverable": ">=0.1.1 <0.3"
-	}
+  "name": "o-footer",
```

Comment:
`bower install …` updates the bower.json file and resets all spacing to spaces instead of tabs.